README.md

```diff
@@ -222,7 +222,7 @@ See e.g. the [`examples/trait-erc20`](https://github.com/paritytech/ink/blob/mas
 
 ### Off-chain Testing
 
-The `#[ink::test]` proc. macro enables off-chain testing. See e.g. the [`examples/erc20`](https://github.com/paritytech/ink/blob/master/examples/erc20/lib.rs#L278-L280) contract on how to utilize those or [the documentation](https://paritytech.github.io/ink/ink_lang/attr.test.html) for details.
+The `#[ink::test]` procedural macro enables off-chain testing. See e.g. the [`examples/erc20`](https://github.com/paritytech/ink/blob/master/examples/erc20/lib.rs#L278-L280) contract on how to utilize those or [the documentation](https://paritytech.github.io/ink/ink_lang/attr.test.html) for details.
```

Comment:
You might want to add `proc-macro` to the whitelist :upside_down_face:

Reply:
@Robbepop What's your opinion: `proc. macro` or `procedural macro`?

Reply:
since it is docs we should avoid having too many abbreviations anyways, so I am in for "procedural macro" tbh.